presto-main/src/main/java/io/prestosql/event/QueryMonitor.java

```diff
@@ -432,37 +430,31 @@ private static void logQueryTimeline(BasicQueryInfo queryInfo)
 
         logQueryTimeline(
                 queryInfo.getQueryId(),
-                queryInfo.getSession().getTransactionId().map(TransactionId::toString).orElse(""),
                 elapsed,
                 elapsed,
                 0,
                 0,
                 0,
-                queryStartTime,
-                queryEndTime);
+                queryStartTime);
     }
 
     private static void logQueryTimeline(
             QueryId queryId,
-            String transactionId,
             long elapsedMillis,
             long planningMillis,
             long schedulingMillis,
             long runningMillis,
             long finishingMillis,
-            DateTime queryStartTime,
-            DateTime queryEndTime)
+            DateTime queryStartTime)
     {
-        log.info("TIMELINE: Query %s :: Transaction:[%s] :: elapsed %sms :: planning %sms :: scheduling %sms :: running %sms :: finishing %sms :: begin %s :: end %s",
+        log.info("TIMELINE: Query %s :: elapsed %sms :: planning %sms :: scheduling %sms :: running %sms :: finishing %sms :: begin %s",
```

Comment:
Is there a value in logging `begin`?

Reply:
After the second thought, I would remove this line completely. If one is interested in such information, then they can use event listener or check out the webui. 

Have you ever used this?